### PR TITLE
 resnet 50 || imagenet || fast.ai + students

### PR DESCRIPTION
**Pull request by jph00**
Updated run with different hyperparams.

### CODE REVIEW

**Comment by brettkoonce** [comment]
🍀

File: ImageNet/train/fastai_pytorch.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    "version": "v1.0",
+    "author": "fast.ai + students team: Jeremy Howard, Andrew Shaw, Brett Koonce, Sylvain Gugger",
+    "authorEmail": "j@fast.ai",
+    "framework": "fastai / pytorch",
+    "codeURL": "https://github.com/fastai/imagenet-fast/blob/1bc5aeec765572397c0ebe4a5d616d03beeeeec1/aws/do_nosh.sh",
+    "model": "Resnet 50",
+    "hardware": "8 * V100 (AWS p3.16xlarge)",
+    "costPerHour": 24.48,
+    "timestamp": "2018-04-20",
+    "misc": {
```
Comment (codyaustun) [inline ImageNet/train/fastai_pytorch.json:11]: What hyperparameters did you use?  Please add the following information to `misc`:

- learning rate schedule
- batch size
- optimizer
- momentum
- weight decay

The [Google ResNet50 submission](https://github.com/stanford-futuredata/dawn-bench-entries/blob/master/ImageNet/train/google_resnet50_tpu_tensorflow18.json) is a good example for reference.

File: ImageNet/train/fastai_pytorch.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
+    "version": "v1.0",
+    "author": "fast.ai + students team: Jeremy Howard, Andrew Shaw, Brett Koonce, Sylvain Gugger",
+    "authorEmail": "j@fast.ai",
+    "framework": "fastai / pytorch",
+    "codeURL": "https://github.com/fastai/imagenet-fast/blob/1bc5aeec765572397c0ebe4a5d616d03beeeeec1/aws/do_nosh.sh",
```
Comment (codyaustun) [inline ImageNet/train/fastai_pytorch.json]: This URL gives a 404 Error. Please correct.

**Review by codyaustun — CHANGES_REQUESTED** [review]

**Comment by jph00** [comment]
@codyaustun - fixed now. Sorry about the incorrect URL! In addition to the hyperparams you requested, I also added the communications backend details to the 'misc' section (note that NCCL 2 isn't open sourced by Nvidia any more so I can't provide a code link for that one.)

**Comment by bignamehyp** [comment]
Did you use the [blacklisted](https://da-data.blogspot.com/2016/02/cleaning-imagenet-dataset-collected.html) validation set?
https://github.com/fastai/imagenet-fast/blob/master/imagenet_nv/blacklist.sh

**Comment by jph00** [comment]
We removed those files as discussed by the devkit notes for the imagenet competition - you've linked to something that claims maybe that it may not the right thing to do, but I believe that the imagenet devkit is fairly clear to the contrary.

(It does mention that the reason for the blacklist was the presence of incorrect bounding boxes, but it does not go on to say to use the blacklisted images for classification models.)

(It would have been better if this was addressed explicitly in the competition rules of course because optimal hyperparams are different depending on whether you do this or not. If different entries are using different approaches then they won't be exactly comparable.)

**Review by codyaustun — APPROVED** [review]

**Comment by codyaustun** [comment]
@jph00 @bignamehyp Aside from the blacklist issue, this pull request looks good to me. Once we decide on the blacklist examples in [issue #36](https://github.com/stanford-futuredata/dawn-bench-entries/issues/36), we will update this submission with the relevant information/fields.

**Comment by jgong5** [comment]
Is this submission using the blacklisted validation set? Then that would not be an apple-to-apple comparison if other teams are not using the same validation set.
